test/Marketplace.js

```diff
@@ -27,10 +28,12 @@ contract("Marketplace2", accounts => {
     describe("Creating, deleting products in market1 and market2", () => {
         //product created in market1
         let id1 = "test1"
+        let id1bytes = web3.utils.padLeft(web3.utils.asciiToHex(id1), 64)
```

Comment:
Truffle5 doesn't accept text strings as bytes32. need to convert to bytes string